openmetadata-ui/src/main/resources/ui/cypress/common/advancedSearch.js

```diff
@@ -392,13 +424,11 @@ export const checkAddGroupWithOperator = (
         .should('be.visible')
         .type(searchCriteria_1);
     } else {
-      interceptURL('GET', '/api/v1/search/suggest?q=*', 'suggestApi');
       cy.get('.widget--widget > .ant-select > .ant-select-selector')
         .eq(index_1)
         .should('be.visible')
         .type(searchCriteria_1);
 
-      verifyResponseStatusCode('@suggestApi', 200);
```

Comment:
Don't we need to intercept API whichever we are using to search?

Reply:
We added this wait as we used the suggest API to get options after searching. But now we are searching internally within the results we get from the Aggregates API, so there is no call for getting suggestions